README.md

```diff
@@ -14,29 +14,30 @@ Our goal with this project is to build an exhaustive collection of awesome resou
 
 -----
 
-- [Papers](#papers)
-  - [Autonomous Task Solver](#autonomous-task-solver)
-    - [General Reasoning & Planning & Tool Using](#general-reasoning--planning--tool-using)
-    - [Multi-Agent Cooperation](#multi-agent-cooperation)
-    - [Framework & Open-Source](#framework--open-source)
-    - [Application](#application)
-      - [Web Agents](#web-agents)
-      - [RL Agents](#rl-agents)
-      - [Robotics & Embodied AI](#robotics--embodied-ai)
-      - [Gaming & Role-Playing](#gaming--role-playing)
-      - [Other Applications](#other-applications)
-    - [Trustworthy](#trustworthy)
-  - [Human Interaction Simulation](#human-interaction-simulation)
-  - [Human-Agent Interaction](#human-agent-interaction)
-  - [Agents-Powered LLMs](#agents-powered-llms)
-  - [Benchmark](#benchmark)
-  - [Survey & Tutorial](#survey--tutorial)
-- [Open-Source Projects](#open-source-projects)
-  - [Autonomous Task Solver Projects](#autonomous-task-solver-projects)
-  - [Multi-Agent Simulation Projects](#multi-agent-simulation-projects)
-- [Perspectives](#perspectives)
-- [Other Related Sources](#other-related-sources)
-- [Acknowledgement](#acknowledgement)
+- [Awesome LLM-Powered Agent](#awesome-llm-powered-agent)
```

Comment:
Is this a necessary modification? Could we roll back to the original table of contents? Thanks!

Reply:
Hi, sorry for the inconvenience. It was an auto update of table contents by VScode when I saved markdown files. I roll back the commit and update with only new papers added. Please have a check.